auction-house/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metaplex-foundation/mpl-auction-house",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
did you actually publish a new version and tag it?
Please do that only once we merged into master since once we squash the PR that tagged commit is no longer there.
Workflow should be:

1. PR changes
2. from master `npm version && npm pub` which tags properly, adds a commit and pushes it + tag